packages/badge/src/Badge.story.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { StoryFn } from '@storybook/react';
 
-import { storybookArgTypes, StoryMetaType } from '@leafygreen-ui/lib';
```

Comment:
You know I've pushed for this name change since day 1, so you also know I am a fan 😎